bridge/elrond/client.go

```diff
@@ -17,13 +18,14 @@ import (
 )
 
 const (
-	ExecutionCost = 1000000000
+	ExecutionCost       = 250000000
```

Comment:
This won't work unfortunately based on costs here https://github.com/ElrondNetwork/sc-bridge-elrond/blob/main/docs/gas_costs.md